server/src/main/java/org/apache/druid/server/coordinator/BalancerStrategy.java

```diff
@@ -71,13 +71,60 @@
    * @return {@link BalancerSegmentHolder} containing segment to move and server it currently resides on, or null if
    *         there are no segments to pick from (i. e. all provided serverHolders are empty).
    */
+  @Deprecated
```

Comment:
this doesn't really seem deprecated so much as no longer called at all, maybe we should remove it?

Reply:
updated